tasks/release_tests.py

```diff
@@ -25,7 +24,7 @@ def test_major_lower(self):
     def test_major_higher(self):
         version = self._get_random_version()
         increment = random.randint(1, 99)
-        self.assertTrue(release._is_version_higher(self._get_version(version["major"] + 1, version["minor"], version["patch"], version["rc"]),
+        self.assertTrue(release._is_version_higher(self._get_version(version["major"] + increment, version["minor"], version["patch"], version["rc"]),
```

Comment:
`increment` was unused, I am guessing the desired behavior was as changed based on the rest of tests.